tests/getFieldDecorator.spec.js

```diff
@@ -190,3 +190,53 @@ describe('dynamic', () => {
     expect(form.getFieldError('title')).toEqual(['Title should be 3+ characters']);
   });
 });
+
+describe('Duplicate field names', () => {
+  class Test extends React.Component {
+    constructor(props) {
+      super(props);
+      this.nameDecorator = this.props.form.getFieldDecorator('title');
+    }
+
+    render() {
+      const { renderDuplicates, form } = this.props;
+      const { getFieldDecorator } = form;
+      return (
+        <div>
+          { getFieldDecorator('title')(<input />) }
+          {
+            renderDuplicates &&
+            <React.Fragment>
+              { this.nameDecorator(<input />) }
```

Comment:
Replace with `getFieldDecorator('title')` in `render`. And test it should not warning.

Reply:
@zombieJ It is what I'm testing on the next line. So, 2 invocations of `getFieldDecorator('title')` is occurring inside render when `renderDuplicates` is true. Won't that satisfy the case you are concerned about?

Reply:
`getFieldDecorator('title')` not `getFieldDecorator('title')(<input />)` (define a field without call it)

Reply:
@zombieJ Done.

Reply:
follow code can be remove now:
```jsx
this.nameDecorator = this.props.form.getFieldDecorator('title');
{ this.nameDecorator(<input />) }
```